src/raftstore/store/worker/compact.rs

```diff
@@ -79,7 +79,7 @@ impl Runner {
         }
         // It is safe to disable WAL here. If crashed, we can still
         // compact the log after restart.
-        box_try!(task.engine.write_without_wal(wb));
+        box_try!(task.engine.write(wb));
```

Comment:
no, it will be removed again after restart. because we seek log in [0, truncated index)